src/js/main-window.js

```diff
@@ -6,9 +6,13 @@ var Accel = require('ui/accel');
 var Utils = require('utils');
 var Menu = require('menu');
 var Elements = require('elements');
+var Feature = require('platform/feature');
+var Settings = require('settings');
+
+Accel.init();
 
 var mainWindow = new UI.Window({
-  fullscreen: true, 
```

Comment:
Weird naming, eh? set status to true if you want fullscreen :) 
When I tested this status flag, it didn't work, but it was half a year ago - maybe they fixed that. Using both properties wouldn't work, as they only look for 'status' if 'fullscreen' is not set.
Ok, let's see. If 'status' works now, then yes - remove the 'fullscreen'